fsi/solutions/unifiedCustomerProfile/readme.md

```diff
@@ -0,0 +1,111 @@
+# Unified Customer Profile
+
+The unified customer profile sceanrio in Microsoft Cloud for Financial Services provides financial institutions with a 360-degree perspective of their customers to enable a more engaged interaction and higher customer satisfaction. The unified profile dashboard can be used across multiple business lines to provide accurate, consistent and cohesive information that can be used to provide a personalized experience, reveal important opportunities and prevent potential loss or churn.
+
+Key capabilitie sof the unified customer profile app include:
+
+* **Customer Summary**: Provides an holistic overview of the customer including their personal details, demographics. life moments, income, financial holdings, credit & debit cards and preferred channel of communication.
+* **Relationships and Groupings**: The app provides information about groups that the customer is part of. Groups can be created and associated based on financial strengths per household or other factors. Relationships between contacts can also be captured and created.
+* **Financial Holdings**: Provides a detailed overview of financial holdings of the respective customer.
+
+| Reference implementation    | Description | Deploy |
+|:----------------------------|:------------|--------|
+| Unified Customer Profile | The scenario enables a holistic overview of a customer in the financial industry. |[![Deploy To Microsoft Cloud](../../../docs/deploytomicrosoftcloud.svg)](https://aka.ms/solutioncenter) |
+
+## Prerequisites
+
+Before you deploy and configure, verify you have implemented the [prerequisites](../../prereqs.md).
+
+Specifically for the unified customer profile scenario, you need:
+
+* [Power Platform environments](../../../foundations/powerPlatform/)
+  * Must be created upfront, in United States with Dataverse and D365 Apps enabled
+
+> Note: The implementation of the unified customer profile solution can currently not be automated, and requires manual implementation and configuration across Solution Center and Power Platform Admin Center.
```

Comment:
Please change to ".... customer profile solution cannot be fully-automated and requires manual intervention to complete configuration steps across Solution Center and Power Platform Admin Center."

Thanks

Reply:
If it's the same across other documents, I think we should consider making it more concise in future. 
Maybe I am being pedantic. Let's chat offline. Approving for now.